thirdeye/thirdeye-pinot/src/test/java/com/linkedin/thirdeye/integration/AbstractDBIntegrationTestBase.java

```diff
@@ -0,0 +1,171 @@
+package com.linkedin.thirdeye.integration;
+
+import java.io.File;
+import java.net.URISyntaxException;
+import java.net.URL;
+import java.sql.Connection;
+import java.sql.ResultSet;
+import java.sql.Statement;
+import java.util.HashSet;
+import java.util.Properties;
+import java.util.Set;
+import java.util.concurrent.TimeUnit;
+
+import javax.persistence.EntityManager;
+
+import org.apache.tomcat.jdbc.pool.DataSource;
+import org.hibernate.cfg.Environment;
+import org.hibernate.engine.jdbc.connections.internal.DatasourceConnectionProviderImpl;
+import org.hibernate.internal.SessionImpl;
+import org.testng.annotations.AfterClass;
+import org.testng.annotations.BeforeClass;
+
+import com.linkedin.thirdeye.common.persistence.PersistenceConfig;
+import com.linkedin.thirdeye.common.persistence.PersistenceUtil;
+import com.linkedin.thirdeye.constant.MetricAggFunction;
+import com.linkedin.thirdeye.datalayer.bao.AnomalyFunctionManager;
+import com.linkedin.thirdeye.datalayer.bao.EmailConfigurationManager;
+import com.linkedin.thirdeye.datalayer.bao.JobManager;
+import com.linkedin.thirdeye.datalayer.bao.MergedAnomalyResultManager;
+import com.linkedin.thirdeye.datalayer.bao.RawAnomalyResultManager;
+import com.linkedin.thirdeye.datalayer.bao.TaskManager;
+import com.linkedin.thirdeye.datalayer.bao.WebappConfigManager;
+import com.linkedin.thirdeye.datalayer.bao.hibernate.AnomalyFunctionManagerImpl;
+import com.linkedin.thirdeye.datalayer.bao.hibernate.EmailConfigurationManagerImpl;
+import com.linkedin.thirdeye.datalayer.bao.hibernate.JobManagerImpl;
+import com.linkedin.thirdeye.datalayer.bao.hibernate.MergedAnomalyResultManagerImpl;
+import com.linkedin.thirdeye.datalayer.bao.hibernate.RawAnomalyResultManagerImpl;
+import com.linkedin.thirdeye.datalayer.bao.hibernate.TaskManagerImpl;
+import com.linkedin.thirdeye.datalayer.bao.hibernate.WebappConfigManagerImpl;
+import com.linkedin.thirdeye.datalayer.dto.AnomalyFunctionDTO;
+import com.linkedin.thirdeye.datalayer.dto.EmailConfigurationDTO;
+
+public abstract class AbstractDBIntegrationTestBase {
```

Comment:
The content of this class is already present in com.linkedin.thirdeye.db.dao.AbstractDbTestBase
We can have better abstraction here.